src/js/utils/playerutils.ts

```diff
@@ -2,13 +2,15 @@ import { version } from 'version';
 import { isFileProtocol } from 'utils/validator';
 
 export const getScriptPath = function(scriptName: string): string {
```

Comment:
return value should be `string | void` yeah?

Reply:
line 15 `return '';`

Reply:
Ugh of course, sorry should have expanded